website/partners/api/v2/views.py

```diff
@@ -18,6 +19,11 @@ class PartnerListView(ListAPIView):
         framework_filters.OrderingFilter,
         framework_filters.SearchFilter,
     )
+
+    def get_serializer(self, partners, *args, **kwargs):
```

Comment:
For some reason the vacancy API does not show thumbnails but rather the full company_logo?